src/common/state-dispatcher.ts

```diff
@@ -28,7 +27,7 @@ export class StateDispatcher {
             this.broadcastMessage({
                 isStoreUpdateMessage: true,
```

Comment:
It looks like there's still a few dangling references to the removed `isStoreUpdateMessage` (here, `state-dispatcher.test.ts`, `store-proxy.test.ts`)